src/Build/BackEnd/Components/SdkResolution/SdkResolution.cs

```diff
@@ -63,10 +63,15 @@ internal SdkResolution(SdkResolverLoader sdkResolverLoader)
                     try
                     {
                         var result = (SdkResultImpl)sdkResolver.Resolve(sdk, context, resultFactory);
-                        if (result != null && result.Success)
+                        if (result == null)
+                        {
+                            continue;
+                        }
+
+                        if (result.Success)
                         {
                             LogWarnings(loggingContext, sdkReferenceLocation, result);
-                            return result.Path;
+                            return result.Path; 
```

Comment:
With @rainersigwald out I'll have to be the one to point out your extra space here :P

Reply:
👍